app/presenters/courses/cohorts_presenter.rb

```diff
@@ -24,6 +24,10 @@ def paged_cohorts
     paged
   end
 
+  def status
+    @status
+  end
```

Comment:
This is used in the view to handle style and status is passed to translations which are shown in the UI.

Based on this https://github.com/pupilfirst/pupilfirst/pull/1379#discussion_r1340410879 comment. We are sticking to show active cohorts instead of crashing the server when an invalid status is searched. The status is useful when a user enters an invalid status in the url

Reply:
@yash-learner It might be idiomatic to use `attr_reader` here. 🤔